lib/awful/screen.lua

```diff
@@ -234,6 +235,63 @@ function screen.object.set_padding(self, padding)
     end
 end
 
+--- A list of outputs for this screen with their size in mm.
+--
+-- This information is only available when RANDR is used to manage the screens.
+--
+-- Please note that the table content may vary.
```

Comment:
What does this mean? Can this text be made more specific and less vague?

The current text could be read as "this table is not always equivalent to `{}`".